CONTRIBUTING.md

```diff
@@ -109,6 +116,19 @@ INCLUDE_ASM(void, "file", func, void);
 #endif
 ```
 
-## NON_MATCHING
+### NON_MATCHING
+
+If you are unable to 100% match a function but think it's equivalent in behavior, you can wrap the C code in NON_MATCHING.
```

Comment:
This should maybe be "certain it's equivalent"?

Reply:
I'll say "fairly certain" just cause I don't wanna scare people off too much. but yeah, stronger than "think" is good